api/client/samples/crop_models/Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3570 @@
+{
```

Comment:
<p>We're almost at the end now! Model failed at this step with this error:</p><p> </p><p><strong>NameError</strong>                                 Traceback (most recent call last)</p><p><strong>&lt;ipython-input-39-d0f15a04a6eb&gt;</strong> in &lt;module&gt;</p><p><strong>----&gt; 1 </strong>with_pred<strong>.</strong>set_index<strong>('end_date')</strong></p><p>      2 </p><p>      3 fig<strong>,</strong> ax <strong>=</strong> plt<strong>.</strong>subplots<strong>(</strong>figsize<strong>=(12,8))</strong></p><p>      4 with_pred<strong>.</strong>plot<strong>(</strong>x<strong>='end_date',</strong> y<strong>='ATR',</strong> ax<strong>=</strong>ax<strong>)</strong></p><p>      5 with_pred<strong>.</strong>plot<strong>(</strong>x<strong>='end_date',</strong> y<strong>='pred',</strong> ax<strong>=</strong>ax<strong>)</strong></p><p> </p><p><strong>NameError</strong>: name 'with_pred' is not defined</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p>fixed</p>